src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -56,7 +60,8 @@ public JsonAdaptedPerson(@JsonProperty("name") String name, @JsonProperty("phone
     public JsonAdaptedPerson(Person source) {
         name = source.getName().fullName;
         phone = source.getPhone().value;
-        email = source.getEmail().value;
+        telegram = source.getTelegram();
+        email = source.getEmail();
```

Comment:
Actually I think we should continue following the original format by using String rather than the Object. Maybe rather than doing this, we can simply let the VALIDATION_REGEX accept "" as well, or just let isValidEmail() accept either "" or test.matches(VALIDATION_REGEX)